tools/generate_plantuml_svg.sh

```diff
@@ -43,4 +45,17 @@ then
     exit
 fi
 
-java -jar $PLANTUML_DIR/plantuml.jar -nometadata -tsvg $EXPORT_DIR/*
+
+# set number of cores for building
+if [[ "$OSTYPE" == "linux-gnu"* ]] || [[ "$OSTYPE" == "cygwin" ]] || [[ "$OSTYPE" == "msys" ]]; then
```

Comment:
Just out of curiosity: How does it take to generate the svg files?

Reply:
How long? About 3 seconds with 12 threads for 12 svg

Reply:
Just in case you want hard numbers ;)
1 thread
```
Executed in    2,98 secs   fish           external 
   usr time    7,39 secs  621,00 micros    7,39 secs 
   sys time    0,23 secs  148,00 micros    0,23 secs
```
12 threads
```
Executed in    1,76 secs   fish           external 
   usr time   10,34 secs  250,00 micros   10,34 secs 
   sys time    0,40 secs   59,00 micros    0,40 secs
```